src/main/java/org/erp/school/controller/HomeController.java

```diff
@@ -1,25 +0,0 @@
-package org.erp.school.controller;
-
-import io.swagger.annotations.Api;
-import org.springframework.security.access.prepost.PreAuthorize;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
-
-import java.security.Principal;
-
-@Controller
-@Api
-public class HomeController {
-
-  @RequestMapping("/helloController")
-  @PreAuthorize("isAuthenticated()")
-  @ResponseBody
-  public String greeting(Principal principal) {
```

Comment:
This is used for the Testing and also to check the authentication via Keycloack. Please, rollback.

Reply:
This shouldn't be used this way. I would suggest switching to actuator endpoints.
Let me know what you think.

Reply:
ok for me.